mbf_msgs/action/ExePath.action

```diff
@@ -8,6 +8,10 @@ string controller
 # use different slots for concurrency
 uint8 concurrency_slot
 
+# define goal tolerance for the mission
+bool goal_tolerance
```

Comment:
Use `tolerance_from_action`, or something similar, otherwise this could be misunderstood.